src/node.cc

```diff
@@ -3751,7 +3752,9 @@ static void ParseArgs(int* argc,
       default_cipher_list = arg + 18;
     } else if (strncmp(arg, "--use-openssl-ca", 16) == 0) {
       ssl_openssl_cert_store = true;
+      use_ca_flag |= 1;
```

Comment:
I don't think `bool a = 1; bool b = 2; if(a && b) ..` is obviously faster than `int i; i |= 1; i |=2; if(i === 3) ...`. If it was hot code, we'd want a benchmark, but its not, clarity should be the goal here.